test/connector.test.js

```diff
@@ -0,0 +1,67 @@
+// Copyright IBM Corp. 2016. All Rights Reserved.
+// Node module: loopback-connector
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const Connector = require('../lib/connector');
+const expect = require('chai').expect;
+const {ModelBuilder} = require('loopback-datasource-juggler');
+
+describe('Connector', function() {
+  describe('getPropertyDefinition()', function() {
+    let connector, builder;
+
+    beforeEach(() => {
+      connector = new Connector('MyConnector');
+      builder = new ModelBuilder();
+
+      const MyModel = builder.define('MyModel', {
+        firstname: String,
+        phoneList: [
+          {
+            number: Number,
+            label: {
+              title: String,
+            },
+          },
+        ],
+        address: {
```

Comment:
I think it'd be good to add a test for an object property with an array within it.
Something like:
```js
someProp: {
  innerArray: [
    {
    someProp: Number
    }
  ]
}
```